BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -37,13 +37,23 @@ public final class BlueprintView: UIView {
     private let rootController: NativeViewController
 
     /// A base environment to render elements with.
-    /// Some keys may be overriden based on the traits of the view itself.
+    /// Some keys may be overridden based on the traits of the view itself.
     public var environment: Environment {
         didSet {
             setNeedsViewHierarchyUpdate()
             invalidateIntrinsicContentSize()
         }
     }
+    
+    /// A base environment inherited when this ``BlueprintView`` is within the view hierarchy of another ``BlueprintView``.
+    /// Set during the view layout portion of a layout cycle.
+    /// Some keys may be overridden based on the traits of the view itself, or based on the explicit environment set on `environment`.
+    public internal(set) var inheritedEnvironment : Environment? {
+        didSet {
+            setNeedsViewHierarchyUpdate()
```

Comment:
Actually; I don't think I need this; since the layout pass itself should do this?